### PR TITLE
[CCTRI-1152] Return already processed data along with occurred error

### PR DESCRIPTION
**Pull request by gdavoian**

### CODE REVIEW

File: tests/functional/tests/test_indicator.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def test_positive_indicator_email(module_headers):
     )['data']
     indicators = get_observables(
         response, 'Have I Been Pwned')['data']['indicators']
-    assert indicators['count'] == 14
+    assert indicators['count'] > 0
```
Comment (mistergreen22) [inline tests/functional/tests/test_indicator.py:27]: Hi, make pls check like this 
`assert len(indicators['docs']) > 0` 
`assert indicators['count'] == len(indicators['docs'])`

File: tests/functional/tests/test_sighting.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def test_positive_sighting_email(module_headers):
     )['data']
     sightings = get_observables(
         response, 'Have I Been Pwned')['data']['sightings']
-    assert sightings['count'] == 14
+    assert sightings['count'] > 0
```
Comment (mistergreen22) [inline tests/functional/tests/test_sighting.py:27]: and here 
`assert len(sightings['docs']) > 0`
`assert sightings['count'] == len(indicators['docs'])`

File: tests/functional/tests/test_indicator.py (hunk 1 above)
Comment (gdavoian) [reply tests/functional/tests/test_indicator.py:27]: Thank you very much Mr. Green! I will definitely do whatever you say!

File: tests/functional/tests/test_sighting.py (hunk 2 above)
Comment (gdavoian) [reply tests/functional/tests/test_sighting.py:27]: Thank you very much Mr. Green! I will definitely do whatever you say!

**Pull request merged**